cmd/encryption-v1.go

```diff
@@ -930,26 +930,29 @@ func (o *ObjectInfo) GetDecryptedRange(rs *HTTPRangeSpec) (encOff, encLength, sk
 	}
 
 	// Assemble slice of (decrypted) part sizes in `sizes`
+	var sizes []int64
 	var decObjSize int64 // decrypted total object size
-	var partSize uint64
-	partSize, err = sio.DecryptedSize(uint64(o.Size))
```

Comment:
This is the exact location of the bug - for a multipart object, this line returns an error and we do not go the next if block that was intended to handle multipart objects. Took me a while to understand the exact problem here.

Reply:
Yes because 

```
encrypted(size_1) + encrypted(size_2) != encrypted(size_1 + size_2) 
```